themes/beaver/assets/css/pages/about-us.css

```
@@ -3008,914 +2941,227 @@ img.mfp-img {
-  text-align: right;
+.pp-rating {
+  text-align: left;
 }
 
 
-@media (max-width: 860px) {
-  .fl-builder-content .fl-node-c17gwsk2h8zy .fl-rich-text, .fl-builder-content .fl-node-c17gwsk2h8zy .fl-rich-text *:not(b, strong) {
-    text-align: center;
-  }
+.pp-rating i {
+  font-style: normal;
```

Stylelint (17.14.0)

[error] Duplicate property "font-style" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)